packages/react-magma-docs/src/pages/design/typography.mdx

```diff
@@ -775,7 +776,7 @@ Another advantage to using the presets above is that Responsive behavior is alre
   <img src="../../images/typography/responsive-example.png" alt=" " />
 </figure>
 <span class="horizontal-spacer"></span>
-<hr class="article-divider" />
+<Divider />
```

Comment:
I was going to say it looked like all the pages in design didn't have `span.horizontal-spacer` and then this one did 🤔

Reply:
Yeah, again, that might not be intentional.  I like the idea of adding the space to the divider.